index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Personal Portfolio</title>
+  <link rel="stylesheet" href="main.css" />
+ 
+</head>
+
+	<body>
+		<header class="Flexbox-header">
+			<div
+				class="flex-container header flex-row align-center justify-between"
+			>
+				<a href="#" class="logo color-primary">My Logo</a>
+				<svg
+					class="NavToggle"
+					width="24"
+					height="24"
+					viewBox="0 0 24 24"
+					fill="none"
+					xmlns="http://www.w3.org/2000/svg"
+				>
```

Comment:
- [ ] I recommend matching the Figma design template as much as possible by adding proper margins.
<table>
<tr>
<th>Your design</th>
<th>Figma</th>
</tr>
  <tr>
    <td><img src="https://github.com/AsciencioAlex/Personal-Portfolio/assets/86069740/db91a8af-e0be-4496-b8b0-f5a0993921d4"/></td>
    <td><img src="https://github.com/AsciencioAlex/Personal-Portfolio/assets/86069740/43edf6c5-2cff-4550-9de8-915df847c6f2"/></td>
  </tr>
</table>